apps/omg_performance/lib/omg_performance/generators.ex

```diff
@@ -61,6 +61,9 @@ defmodule OMG.Performance.Generators do
 
   Options:
     - :use_blocks - if not nil, will use this as the stream of blocks, otherwise streams from child chain rpc
+    - :no_deposit_spends - if true, will limit only to transactions that do ont spend deposits (see function with the
+      same name for explanation)
+    - :sent_by - if not nil, will limit to txs sent
```

Comment:
hm... elaborate what this flag does?

Reply:
whoops, part of the sentence chopped off